manifests/overlays/prod/secrets/clusters/secret-generator.yaml

```diff
@@ -4,5 +4,7 @@ metadata:
   name: cluster-spec-generator
 files:
 - secrets/clusters/apps.ocp.prod.psi.redhat.com.enc.yaml
+- secrets/clusters/api.aasthana.dev.datahub.redhat.com.enc.yaml
```

Comment:
Was this change supposed to be part of this PR?

Reply:
yep!

Reply:
(we forgot to add this line in a previous PR)